lib/Stampie/MailerInterface.php

```diff
@@ -12,12 +12,12 @@
 interface MailerInterface
 {
     /**
-     * @param AdapterInterface $adapter
+     * @param HttpClient $adapter
      */
-    function setAdapter(AdapterInterface $adapter);
+    function setAdapter(HttpClient $adapter);
 
     /**
-     * @return AdapterInterface
+     * @return HttpClient
      */
     function getAdapter();
```

Comment:
@henrikbjorn if we make a new major release, do we really need to have such methods in the interface ? IMO, the HTTP client should be considered an implementation detail rather than being part of the interface